src/content/docs/en/guides/images.mdx

```diff
@@ -159,6 +159,37 @@ import myImage from '../assets/my_image.png'; // Image is 1600x900
 
 <ReadMore>See details about [the `<Picture />` component properties](/en/reference/modules/astro-assets/#picture-properties) in the `astro:assets` reference.</ReadMore>
 
+### Inline `.svg` files in the HTML
+
+<p><Since v="5.0.0" /></p>
+
+To inline local `.svg` files directly in your HTML, SVG files can be imported and used like any other [Astro components](/en/basics/astro-components/#component-based-design).
+
+```astro
+---
+import Logo from '../assets/logo.svg';
+---
+
+<Logo />
```

Comment:
Question: what happens if no attributes are passed? Is meta data from the `.svg` file used so that the image is basically presented as is? Are there any required attributes, or will this just work like this?

I'm not sure it's entirely necessary to actually state more here as long as someone can literally just do this and get SOME kind of image (even if not the way they want it to eventually look). Just want to make sure nothing is required, and everything else is an optional tweak of the logo file.

Reply:
If nothing is passed it will just use any attributes that are on the SVG file. I think this is the expected functionality.